store/src/java/com/zimbra/cs/account/ZAttrCos.java

```diff
@@ -20182,6 +20182,78 @@ public Map<String,Object> unsetGalSyncAccountBasedAutoCompleteEnabled(Map<String
         return attrs;
     }
 
+    /**
+     * When set to True, all aliases will be hidden for particular account
+     *
+     * @return zimbraHideAliasesInGal, or false if unset
+     *
+     * @since ZCS 10.1.0
```

Comment:
below at many places, it should be 10.2.0

Reply:
fixed